src/ui/templates/cards/accordion.hbs

```diff
@@ -1,26 +1,36 @@
 <div class="yxt-AccordionCard js-yxt-AccordionCard{{#if isExpanded}} yxt-AccordionCard--expanded{{/if}}"
-  id="{{id}}">
-  <button class="yxt-AccordionCard-toggle js-yxt-AccordionCard-toggle"
-    aria-expanded="{{#if isExpanded}}true{{else}}false{{/if}}"
-    aria-controls="{{id}}"
-    tabindex="0"
-  >
+id="{{id}}">
+
+  {{#if _config.details}}
+    <button class="yxt-AccordionCard-toggle js-yxt-AccordionCard-toggle"
+      aria-expanded="{{#if isExpanded}}true{{else}}false{{/if}}"
+      aria-controls="{{id}}"
+      tabindex="0"
+    >
+      <h3 class="yxt-AccordionCard-title">
+        {{{_config.title}}}
+      </h3>
+      <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}"
+        data-component="IconComponent"
+        data-opts='{"iconName": "chevron"}'
+        data-prop="icon">
+      </div>
+    </button>
+  {{else}}
     <h3 class="yxt-AccordionCard-title">
       {{{_config.title}}}
     </h3>
-    <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}"
-      data-component="IconComponent"
-      data-opts='{"iconName": "chevron"}'
-      data-prop="icon">
+  {{/if}}
+
+  {{#if _config.details}}
+    <div class="yxt-AccordionCard-body js-yxt-AccordionCard-body" aria-hidden="{{#if isExpanded}}false{{else}}true{{/if}}">
+      <div class="yxt-AccordionCard-content">
+        <div class="yxt-AccordionCard-subtitle">{{{_config.subtitle}}}</div>
```

Comment:
I realize that what I am asking is a breaking change. Maybe we want an item for this for later? config.subtitle and config.title, are they guaranteed to be available? Do we want to check for their existence before showing the html (which have margin/padding styling)?

Reply:
It's been unguarded like that for a while. Made an item to address in a subsequent release: https://yexttest.atlassian.net/browse/SPR-2445